From 53ab64f20e10e12d829ab670267c7390638ebc72 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 04 Dec 2018 18:14:53 +0200
Subject: [PATCH] SPL copy prepare: remove note that does not hold

Remove the comment note that does not hold anymore as we hard code
the size of SPL in define.
No functional changes.

Change-Id: Ifbf17ec0da22ca2b3461f87fda1e8290bf04c89f
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/preboot.c b/preboot.c
index f3d3b93..22dcf11 100644
--- a/preboot.c
+++ b/preboot.c
@@ -104,7 +104,6 @@
 /*
  * Using scratchpad registers for
  * passing flash layout to CPU SW
- * Note: keep images size set by BTL
  */
 static void hl_preboot_copy_prepare(unsigned int *spl_off)
 {
